### DISCUSSION TITLE
Partially Watched Shows/Movies?

### DISCUSSION THREAD

**Discussion by StrangerJona** [arabcoders/watchstate#552] [General] [answered]
Wondering if watchstate can sync the time of a show/movie at which point it was stopped on either plex or jellyfin, as a Continue Watching entry?

**Comment by arabcoders** [comment, marked as answer]
Hello,

Sorry for the late reply i didn't notice it. Yes it does sync partially watched content, we call it "progress sync", However plex does have problem if you didn't start watching the content i.e. the show didn't have at least one episode watched it will sync the progress but won't show up in the continue watching
